src/quacc/recipes/newtonnet/core.py

```diff
@@ -0,0 +1,399 @@
+from monty.dev import requires
+
+try:
+    from sella import IRC, Sella
+except:
+    Sella = None
+from quacc import SETTINGS
+from quacc.schemas.ase import (
+    summarize_opt_run,
+    summarize_run,
+    summarize_thermo_run,
+    summarize_vib_run,
+)
+from quacc.util.calc import run_ase_opt, run_calc
+from quacc.util.thermo import ideal_gas
+
+try:
+    from newtonnet.utils.ase_interface import MLAseCalculator as NewtonNet
+except ImportError:
+    NewtonNet = None
+
+
+def get_hessian(atoms):
+    mlcalculator = NewtonNet(
+        model_path=SETTINGS.NEWTONNET_MODEL_PATH.split(":"),
+        settings_path=SETTINGS.NEWTONNET_CONFIG_PATH.split(":"),
+    )
+    mlcalculator.calculate(atoms)
+    return mlcalculator.results["hessian"].reshape((-1, 3 * len(atoms)))
+
+
+def add_stdev_and_hess(summary):
+    for i in range(len(summary["trajectory"])):
+        mlcalculator = NewtonNet(
+            model_path=SETTINGS.NEWTONNET_MODEL_PATH.split(":"),
+            settings_path=SETTINGS.NEWTONNET_CONFIG_PATH.split(":"),
+        )
+        mlcalculator.calculate(summary["trajectory"][i]["atoms"])
+        summary["trajectory_results"][i]["hessian"] = mlcalculator.results["hessian"]
+        summary["trajectory_results"][i]["energy_std"] = mlcalculator.results[
+            "energy_disagreement"
+        ]
+        summary["trajectory_results"][i]["forces_std"] = mlcalculator.results[
+            "forces_disagreement"
+        ]
+        summary["trajectory_results"][i]["hessian_std"] = mlcalculator.results[
+            "hessian_disagreement"
+        ]
+    return summary
+
+
+@ct.electron
+@requires(NewtonNet, "NewtonNet must be installed. Try pip install quacc[newtonnet]")
+def static_job(
+    atoms: Atoms, newtonnet_kwargs: dict | None = None, opt_swaps: dict | None = None
+) -> dict:
+    """
+    Carry out a single-point calculation.
+
+    Parameters
+    ----------
+    atoms : Atoms
+        The atomic configuration to be relaxed.
+    newtonnet_kwargs : dict, optional
+        Additional keyword arguments for the tblite calculator. Defaults to None.
+
+    Returns
+    -------
+    dict
+        A summary of the run, including relevant information about the calculation results.
+    """
+    newtonnet_kwargs = newtonnet_kwargs or {}
+    opt_swaps = opt_swaps or {}
+    input_atoms = deepcopy(atoms)
+    # Define calculator
+    mlcalculator = NewtonNet(
+        model_path=SETTINGS.NEWTONNET_MODEL_PATH.split(":"),
+        settings_path=SETTINGS.NEWTONNET_CONFIG_PATH.split(":"),
+    )
+    atoms.calc = mlcalculator
+    atoms = run_calc(atoms)
+    return summarize_run(
+        atoms, input_atoms=input_atoms, additional_fields={"name": "NewtonNet Relax"}
+    )
+
+
+@ct.electron
+@requires(
+    NewtonNet,
+    "newtonnet must be installed. Checkout https://github.com/ericyuan00000/NewtonNet",
+)
+def relax_job(
+    atoms: Atoms,
+    fmax: float = 0.01,
+    max_steps: int = 1000,
+    optimizer: Optimizer = Sella,
```

Comment:
This line will cause a crash if `Sella` is not installed.

Reply:
Just a reminder that when custom_hessian is asked by the user, Sella is the only optimizer that can do TS optimization in our implementation.

Reply:
Maybe you can do something like Sam did here: https://github.com/Quantum-Accelerators/quacc/pull/393/files#diff-bd8364980ac1eda5810606c2a610911778b2e51df2db8ee501c4f0494cd8398dR194-R199

Reply:
You can do something like Sam did here: https://github.com/samblau/quacc/blob/b4899388f51ec20b42549748802b9f88bd4b64ba/src/quacc/recipes/qchem/core.py#L194-L199

Reply:
@kumaranu: Ah, I see your comment now. Then perhaps we should require the user to have Sella installed. If that's the case, you'll want to add that to the `monty.dev.requires` decorator.